frameworklauncher/src/main/java/com/microsoft/frameworklauncher/common/model/AclConfiguration.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation
+// All rights reserved. 
+//
+// MIT License
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated 
+// documentation files (the "Software"), to deal in the Software without restriction, including without limitation 
+// the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and 
+// to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+// The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING 
+// BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND 
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, 
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, 
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE. 
+
+package com.microsoft.frameworklauncher.common.model;
+
+import com.microsoft.frameworklauncher.common.validation.MapKeyNamingValidation;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+public class AclConfiguration implements Serializable {
+  @Valid
+  @NotNull
+  @MapKeyNamingValidation
+  // Namespace -> AccessControlList
+  // Namespace can be UserName, GroupName, etc.
+  private Map<String, AccessControlList> namespaceAcls = new HashMap<>();
+
+  public Map<String, AccessControlList> getNamespaceAcls() {
+    return namespaceAcls;
+  }
+
+  public void setNamespaceAcls(Map<String, AccessControlList> namespaceAcls) {
+    this.namespaceAcls = namespaceAcls;
+  }
+
+  public AccessControlList getNamespaceAcl(String namespace) {
+    return namespaceAcls.getOrDefault(namespace, new AccessControlList());
```

Comment:
>new AccessControlList() [](start = 49, length = 23)

This is not a good method, it should let external to handle it immutably or mutably.